packages/fxa-auth-server/bin/key_server.js

```diff
@@ -104,6 +105,15 @@ async function run(config) {
     Container.get(PlayBilling);
   }
 
+  // Create AppleIAP if enabled by fetching it.
+  if (
+    config.subscriptions &&
+    config.subscriptions.appStore &&
+    config.subscriptions.appStore.enabled
```

Comment:
Optional chaining?

Reply:
Sure; I copied the PlayBilling logic, but I can do this instead:
```js
  if (config?.subscriptions?.appStore?.enabled) {
    Container.get(AppleIAP);
  }
```